### PR TITLE
react_autocomplete

### PR DESCRIPTION
**Pull request by georgiyevv**
[DEMO LINK](https://svyatoslavPy.github.io/react_autocomplete/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,73 +1,92 @@
...
+  const handlerOnSelected = (person: Person) => {
+    setSelectedPerson(person);
+    setShowPeople(false);
+  };
+
+  const filteredPeople = peopleFromServer.filter((people: Person) => {
+    return people.name.includes(debounceQuery);
+  });
+
+  const person = `${selectedPerson?.name} (${selectedPerson?.born}) - (${selectedPerson?.died})`;
```
Comment (vadiimvooo) [inline src/App.tsx:62]: This text should be in format: `Maria Sturm (1835 - 1917)`

File: src/App.tsx [hunk 2]
```diff
@@ -1,73 +1,92 @@
...
             <input
+              ref={searchInputRef}
+              onClick={() => setShowPeople(true)}
+              onChange={handlerChangeQuery}
               type="text"
+              value={query}
               placeholder="Enter a part of the name"
               className="input"
               data-cy="search-input"
             />
```
Comment (vadiimvooo) [inline src/App.tsx:82]: You should remove dropdown on blur.

File: src/App.tsx [hunk 3]
```diff
@@ -1,73 +1,92 @@
...
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Petronella de Decker</p>
-              </div>
-
-              <div className="dropdown-item" data-cy="suggestion-item">
-                <p className="has-text-danger">Elisabeth Hercke</p>
-              </div>
-            </div>
-          </div>
+          {showPeople && !!filteredPeople.length && (
```
Comment (vadiimvooo) [inline src/App.tsx:84]:
```suggestion
          {showPeople && filteredPeople.length && (
```

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Good job, but try to pass all test. Here are a few fixes which can help you with it. Also feel free to ask for help in fe_chat if you need it.

**Review by ghost — APPROVED** [review]
Good job!

**Pull request closed without merging**